kolibri_instant_schools_plugin/sms/utils.py

```diff
@@ -0,0 +1,41 @@
+import os
+from twilio.rest import Client
+from django.core.urlresolvers import reverse
+
+from ..auth.mapping import normalize_phone_number
+
+COUNTRY_PHONE_PREFIX = os.environ["COUNTRY_PHONE_PREFIX"]
+SMS_MESSAGE_TEMPLATE = os.environ["SMS_MESSAGE_TEMPLATE"]
+TWILIO_SID = os.environ["TWILIO_SID"]
+TWILIO_AUTH_TOKEN = os.environ["TWILIO_AUTH_TOKEN"]
+
+
+def send_password_reset_link(phone, token, baseurl):
+
+    phone = normalize_phone_number(phone)
+    baseurl = baseurl.rstrip("/")
+    endpointurl = reverse("kolibri:kolibri_instant_schools_plugin:passwordchange_list")
+    path = "{endpointurl}{phone}/{token}".format(endpointurl=endpointurl, token=token, phone=phone)
+
+
+    url = "{baseurl}/{path}".format(
+        baseurl=baseurl, path=path
+    )
+
+    send_message(phone, SMS_MESSAGE_TEMPLATE.format(url=url))
+
+
+def send_message(phone, message):
+
+    client = Client(TWILIO_SID, TWILIO_AUTH_TOKEN)
+
+    if not phone.startswith(COUNTRY_PHONE_PREFIX):
+        phone = COUNTRY_PHONE_PREFIX + phone
```

Comment:
I guess keeping this would prevent someone from another country (e.g. Chris or us, testing) from using the password reset flow with our phones, but that's not the end of the world.

Reply:
And better to have it be resilient for those actually within the country.